qcengine/programs/tests/test_molpro.py

```diff
@@ -22,7 +22,7 @@ def test_molpro_output_parser(test_case):
     data = molpro_info.get_test_data(test_case)
```

Comment:
It looks like qcer doesn't populate the list `test_case` (line 18) for Molpro and Terachem on the CI build. It does run for me locally.